task/fbc-validation/0.1/fbc-validation.yaml

```diff
@@ -121,27 +121,32 @@ spec:
         # copy content of conffolder to confdir - will be used in next task - related image check
         cp -r .$conffolder/* $(workspaces.workspace.path)/hacbs/$(context.task.name)/confdir
 
-        # We have totally 4 checks here currently
-        check_num=4
+        EXTRACT_DIR="/extracted_base_img"
+        mkdir "${EXTRACT_DIR}"
+        oc image extract ${BASE_IMAGE} "${EXTRACT_DIR}"
+        OPM_BINARIES="$(find ~+${EXTRACT_DIR} -type f -name opm)"
+        if [[ $(wc -l <<< "$${OPM_BINARIES}") -ne "1" ]]; then
```

Comment:
$$ returns process id, so the line is always 1, I think changing it to "${OPM_BINARIES}" will do the trick.